Description: Fix compiler and linker flags in Makefile.
 The Makefile of wmtime had to be patched in order for CFLAGS
 not to be redefined by the Makefile but to use the CFLAGS
 passed by debhelper and add additional flags through
 string concatenation. LDFLAGS and CPPFLAGS weren't passed
 at all and have been added to the linker command line.
 .

-- 

Author:  John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Bug-Debian: (none)
Applied-Upstream: (none)

---

Index: wmtime-1.0b2/wmtime/Makefile
===================================================================
--- wmtime-1.0b2.orig/wmtime/Makefile	2013-09-21 20:32:56.000000000 +0200
+++ wmtime-1.0b2/wmtime/Makefile	2013-09-21 20:43:10.386769250 +0200
@@ -7,13 +7,13 @@
 XPMS = wmtime-master.xpm wmtime-mask.xbm
 
 CC = gcc
-CFLAGS = -O2
+CFLAGS += -O2
 
 .c.o:
-	$(CC) $(CFLAGS) -c -Wall $< -o $*.o
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -c -Wall $< -o $*.o
 
 wmtime: $(OBJS) $(XPMS)
-	$(CC) $(CFLAGS) -o wmtime $(OBJS) $(LIBDIR) $(LIBS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -o wmtime $(OBJS) $(LIBDIR) $(LIBS)
 
 clean::
 	for i in $(OBJS) ; do \
